activities/TamTamMicro.activity/css/activity.css

```diff
@@ -20,6 +24,135 @@
   z-index: 5;
 }
 
+/*Tutorial CSS*/
+.tutorial-title {
+	background-color: #808080 !important;
```

Comment:
Could you avoid mixing different type of indentation? Here indentation should be white spaces.